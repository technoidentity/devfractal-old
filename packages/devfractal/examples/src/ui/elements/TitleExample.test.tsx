import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Title } from './TitleExample'

test('Title Example', () => {
  const component: ReactTestRenderer = renderer.create(<Title />)
  expect(component.toJSON()).toMatchSnapshot()
})
