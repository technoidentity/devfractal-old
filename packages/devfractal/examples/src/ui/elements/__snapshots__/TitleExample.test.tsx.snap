// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Title Sizes
    </h1>
    <div
      className="box"
    >
      <h1
        className="title is-1"
      >
        Title 1
      </h1>
      <h1
        className="title is-2"
      >
        Title 2
      </h1>
      <h1
        className="title is-3"
      >
        Title 3(default size)
      </h1>
      <h1
        className="title is-4"
      >
        Title 4
      </h1>
      <h1
        className="title is-5"
      >
        Title 5
      </h1>
      <h1
        className="title is-6"
      >
        Title 6
      </h1>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Subtitle Sizes
    </h1>
    <div
      className="box"
    >
      <h1
        className="subtitle is-1"
      >
        Subtitle 1
      </h1>
      <h1
        className="subtitle is-2"
      >
        Subtitle 2
      </h1>
      <h1
        className="subtitle is-3"
      >
        Subtitle 3
      </h1>
      <h1
        className="subtitle is-4"
      >
        Subtitle 4
      </h1>
      <h1
        className="subtitle is-5"
      >
        Subtitle 5
      </h1>
      <h1
        className="subtitle is-6"
      >
        Subtitle 6
      </h1>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Combined Example
    </h1>
    <div
      className="box"
    >
      <h1
        className="title is-1"
      >
        Title 1
      </h1>
      <h1
        className="subtitle is-3"
      >
        Subtitle 3
      </h1>
      <h1
        className="title is-2"
      >
        Title 2
      </h1>
      <h1
        className="subtitle is-4"
      >
        Subtitle 4
      </h1>
      <h1
        className="title is-3"
      >
        Title 3
      </h1>
      <h1
        className="subtitle is-5"
      >
        Subtitle 5
      </h1>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      With Spacing
    </h1>
    <div
      className="box"
    >
      <h1
        className="title is-1 is-spaced"
      >
        Title 1
      </h1>
      <h1
        className="subtitle is-3"
      >
        Subtitle 3
      </h1>
      <h1
        className="title is-2 is-spaced"
      >
        Title 2
      </h1>
      <h1
        className="subtitle is-4"
      >
        Subtitle 4
      </h1>
      <h1
        className="title is-3 is-spaced"
      >
        Title 3
      </h1>
      <h1
        className="subtitle is-5"
      >
        Subtitle 5
      </h1>
    </div>
  </section>,
]
`;
